src/helpers/loadENV.js

```diff
@@ -0,0 +1,3 @@
+window.location.hostname !== 'dashboard.realdevsquad.com'
+  ? (window.API_BASE_URL = 'https://staging-api.realdevsquad.com')
+  : (window.API_BASE_URL = 'https://api.realdevsquad.com');
```

Comment:
@akshay1502 does this piece of code ring some bell for some feature? 😄

Reply:
Yup. For staging and production login right?